book/07-git-tools/sections/rerere.asc

```diff
@@ -1,31 +1,31 @@
 [[_rerere]]
-=== Rerere
+=== Ang Rerere
 
-The `git rerere` functionality is a bit of a hidden feature.
-The name stands for ``reuse recorded resolution'' and, as the name implies, it allows you to ask Git to remember how you've resolved a hunk conflict so that the next time it sees the same conflict, Git can resolve it for you automatically.
+Ang `git rerere` pag-andar ay maliit na nakatagong tampok.
+Ang pangalan ay nakatayong ``reuse recorded resolution'' at, bilang nagpapahiwatig na pangalan, pinapayagan ka nitong hilingan ang Git na tandaan kung paano mo nalutas ang isang malaking naputol na pagsasalungat kaya sa susunod na makakita ito ng parehong pagkasalungat, ang Git ay maaaring malutas ito para sa iyo ng awtomatiko.
 
-There are a number of scenarios in which this functionality might be really handy.
-One of the examples that is mentioned in the documentation is when you want to make sure a long-lived topic branch will ultimately merge cleanly, but you don't want to have a bunch of intermediate merge commits cluttering up your commit history.
-With `rerere` enabled, you can attempt the occasional merge, resolve the conflicts, then back out of the merge.
-If you do this continuously, then the final merge should be easy because `rerere` can just do everything for you automatically.
+Mayroong mga bilang ng mga sitwasyon na kung saan ang pag-andar nito ay maaaring talagang magaling.
```

Comment:
`maaaring talagang` to `maaari talagang`

Reply:
done change